packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js

```diff
@@ -54,14 +36,32 @@ const CMEditViewCopyLocale = ({ appLocales, currentLocale, localizations }) => {
     const requestURL = `/content-manager/collection-types/${slug}/${value.value}`;
 
     try {
+      setIsLoading(true);
       const response = await request(requestURL, { method: 'GET' });
-      const cleanedData = cleanData(response, allLayoutData);
+      const cleanedData = cleanData(response, allLayoutData, localizations);
 
       dispatch({ type: 'ContentManager/CrudReducer/GET_DATA_SUCCEEDED', data: cleanedData });
 
-      handleToggle();
+      strapi.notification.toggle({
```

Comment:
@mfrachet do you want me to split this function?

Reply:
Yes if possible, I was thinking of something like:

```js
const copyLocale = useCopyLocale()
```

and then, you can call the `copyLocale` function inside `handleConfirmCopyLocale` one